tests/test_core.py

```diff
@@ -512,6 +512,23 @@ def test_dagobah_expand_simple_job():
 
 
 @with_setup(blank_dagobah)
+def test_dagobah_expand_same_job_twice():
+    dagobah.add_job('test_job_a')
```

Comment:
It looks like you are testing successful conditions. I think you should also test some failure conditions as well.